JS/js-variables-and-numbers_simple-calculator/js/index.js

```diff
@@ -61,24 +74,42 @@ const divideByTwoButton = document.querySelector('[data-js="divide-by-two"]');
 
 increaseByOneButton.addEventListener("click", () => {
   // 7. Increase the value of "operand1" by 1. Log the new value to the console.
+  let operand1Upd = 12;
+  operand1Upd++;
+  console.log(operand1Upd);
 });
 
 increaseByFiveButton.addEventListener("click", () => {
   // 8. Increase the value of "operand1" by 5. Log the new value to the console.
+  let operand1Upd = 12;
+  operand1Upd += 5;
```

Comment:
We are asking to do it with operand1 that is declare in line 3. Not to declare a new variable.